### PR TITLE
Enable JIT in workflows

### PR DESCRIPTION
**Pull request by taperropein**
Tested locally with [act](https://nektosact.com/) as much as I could, since Github wasn't running it.  Seems to be working.

edit: Of course the ARM tests that I couldn't run locally failed in CI.  Ah well.

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -209,7 +214,8 @@ jobs:
     - name: Run tests
       run: |
         python tools\run-tests.py --engine=%cd%\out\walrus.exe
-      shell: cmd
+        python tools\run-tests.py --jit --engine=%cd%\out\walrus.exe
+     shell: cmd
```
Comment (zherczeg) [inline .github/workflows/actions.yml:218]: Why did the indentation change?

Comment (taperropein) [reply .github/workflows/actions.yml]: Typo, basically.  Fixed it now.

**Comment by taperropein** [comment]
Looks like everything runs, and luckily only one platform has a bug (that the current test suite discovers): Windows x64 with JIT.

**Comment by zherczeg** [comment]
It is stiil a progress. We should not enable Windows 64 then at the moment until we understand what is happening there.

**Comment by taperropein** [comment]
I disabled the JIT tests on x64 Windows for now, plus added a message so we know it's disabled.

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -209,6 +214,8 @@ jobs:
     - name: Run tests
       run: |
         python tools\run-tests.py --engine=%cd%\out\walrus.exe
+        echo 'not running jit tests'
+        rem python tools\run-tests.py --jit --engine=%cd%\out\walrus.exe
```
[the remark this answers is not in the document]
Comment (taperropein) [reply .github/workflows/actions.yml]: Done.

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**